forge-gui/res/cardsfolder/upcoming/lurker_in_the_deep.txt

```diff
@@ -0,0 +1,13 @@
+Name:Lurker in the Deep
+ManaCost:3 U U U
+Types:Enchantment Creature Fish Illusion
+PT:7/7
+K:Impending:4:2 U U
+T:Mode$ ChangesZone | Destination$ Battlefield | Origin$ Any | ValidCard$ Card.Self | Execute$ TrigSeek | TriggerDescription$ Whenever CARDNAME enters or attacks, seek a nonland card.
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigSeek | Secondary$ True | TriggerDescription$ Whenever CARDNAME enters or attacks, seek a nonland card.
+SVar:TrigSeek:DB$ Seek | Type$ Card.nonLand | RememberFound$ True
```

Comment:
this won't work
instead you need to extend the call to SeekAll trigger inside SeekEffect so it also passes the soughtCards